test/_unit.js

```diff
@@ -201,12 +201,12 @@ suite('Lexer', () => {
         });
       });
       Object.keys(external4).forEach(function (k) {
-        if (k !== 'default') exports[k] = external4[k];
+        if (k !== 'default' && !Object.hasOwnProperty.call(exports, k)) exports[k] = external4[k];
```

Comment:
Maybe it's better to keep both tests, to ensure compatibility with older Rollup versions.

Reply:
Good point.